modules/goldilocks/README.md

```diff
@@ -0,0 +1,26 @@
+# todo
+- Goldilocks is a Kubernetes controller that provides a dashboard that gives recommendations on how to set your resource requests.
+
+## Usage
+
+# Case 1
+
+The module create Goldilocks installation prerequisites. The module default create the prerequisites.
+
+° vertical-pod-autoscaler configure in the cluster
+° metrics-server 
+
+```
+module "goldilocks" {
```

Comment:
have an minimal params passing example also,
and run terraform-docs to generate README rest standard documentation part:
```sh
terraform-docs markdown table --output-file README.md --output-mode inject ./modules/goldilocks
```